src/components/OffCanvas/OffCanvasBody.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import * as baseTheme from './OffCanvas.scss';
+import {classNames} from '@shopify/react-utilities/styles';
+
+export interface State{
+    isMenuOpened:false
+}
+export interface Props{
+    width?:number,
+    transitionDuration?:number,
+    isMenuOpened?:boolean,
+    position?:any,    
+    children?:React.ReactNode,    
+    style?:any,
+    animation?:string
+}
+
+export class OffCanvasBody extends React.PureComponent<Props,State>
+{
```

Comment:
bracket should be the last character on the previous line with a space between it and the closing >